crates/bevy_ecs/src/schedule/state.rs

```diff
@@ -683,4 +689,30 @@ mod test {
         stage.run(&mut world);
         assert!(*world.get_resource::<bool>().unwrap(), "after test");
     }
+
+    #[test]
+    fn pr_2211() {
```

Comment:
could you give the test a name that describe what it's testing?

Reply:
```suggestion
    fn repeated_on_in_stack_triggers() {
```